Library/Homebrew/cmd/install.rb

```diff
@@ -263,6 +263,14 @@ def run
           end
         end
 
+        if Homebrew::Attestation.enabled?
+          if formulae.include?(Formula["gh"])
+            formulae.unshift(T.must(formulae.delete(Formula["gh"])))
```

Comment:
Just making sure I understand the behavior here: this is pushing `gh` onto the front of the formulae list, right? Even if it was in some other position?

Reply:
That is correct. If `gh` is present anywhere in the list, this would move it to the head of the list so that it is installed first.